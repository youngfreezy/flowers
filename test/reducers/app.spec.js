import reducer from 'reducers/app';

describe('App', () => {
  const app = reducer.app(undefined, {});

  it('should return the initial state', () => {
    expect(reducer.app(app, {})).toMatchSnapshot();
  });
});
