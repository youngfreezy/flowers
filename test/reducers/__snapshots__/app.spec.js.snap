// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should return the initial state 1`] = `
Object {
  "alerts": Array [],
}
`;
